backend/index.js

```diff
@@ -8,47 +8,12 @@ const cluster = require('./helpers/cluster');
 const joinUs = require('./app/models/joinUs');
 const contactUs = require('./app/models/contactUs');
 const sendEmail = require('./utility/sendEmail');
-const Resource = require('./app/models/resource');
 const {
   JoinUsCronJobMailTemplate,
   ContactUsCronJobMailTemplate,
-  ResourceDeletedMailTemplate,
 } = require('./utility/emailTemplates');
 const Admin = require('./app/models/Admin');
 
-// Cron job to delete all resource data for 2months
-cronJob('0 0 2 * *', async (req, res, next) => {
```

Comment:
we might need the cron job later to delete the broadcast/contactUs data which are older than 6 months/1 year as they will just keep on accumulating no?, for now it should be fine

Reply:
Since there was no cron job for the broadcast, I didn't add it. Can I add it to the current PR?